command/git.go

```diff
@@ -573,3 +580,15 @@ func (c *gitFetchProject) applyPatch(ctx context.Context, logger client.LoggerPr
 	}
 	return nil
 }
+
+type noopWriteCloser struct {
+	w *bytes.Buffer
+}
+
+func (c noopWriteCloser) Write(data []byte) (n int, err error) {
+	return c.w.Write(data)
+}
```

Comment:
I think if you make the *bytes.Buffer anonymous, then you don't have to implement this. 

also avoid declaring n and err in the signature...

Reply:
fixed